app/models/proxies/government_proxy.rb

```diff
@@ -0,0 +1,34 @@
+require 'json'
+
+module Proxies
+  class GovernmentProxy < Rack::Proxy
```

Comment:
Maybe something like `IframeAllowingProxy` would reveal the intent? We could also add a few comments on why we're doing this.